src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,175 @@
+Therefore, [v5](https://docs.giantswarm.io/api/#operation/addClusterV5) API is used now within happa and [gsctl](https://docs.giantswarm.io/reference/gsctl/create-cluster/).
+
+As for now, Giant Swarm is moving towards replacing its own API with the direct cluster management via [Control Plane Kubernetes API](https://docs.giantswarm.io/basics/aws-architecture/#giant-swarm-control-plane).
+Following this movement strategy, Giant Swarm API is going to be deprecated in the near feature. You can find related roadmap issue [here](https://github.com/giantswarm/roadmap/issues/90).
+
+## How does cluster creation works now?
+
+All the tenant clusters, created with release version 10.x.x+, are managed as [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in Control Plane Kubernetes.
+That means, when you're creating a new cluster via [v5](https://docs.giantswarm.io/api/#operation/addClusterV5) API, what *API* service does, it uses [Cluster API](https://github.com/kubernetes-sigs/cluster-api) to create tenant Kubernetes control-plane CR and optional node pools CRs.
+The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style APIs to cluster creation, configuration, and management. It provides optional, additive functionality on top of core Kubernetes.
+
+On high-level Cluster API is used to manage two types of CR kinds:
+  - Cluster
+  - MachineDeployment
+
+## Cluster CRs
+
+*Cluster* CR represents tenant cluster Kubernetes control plane.
+
+*Cluster* CR object example :
+
+```
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  generation: 1
+  labels:
+    cluster-operator.giantswarm.io/version: 2.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: nzr5z
+  namespace: default
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.giantswarm.io/v1alpha2
+    kind: AWSCluster
+    name: nzr5z
+    namespace: default
+```
+
+*Cluster* references *AWSCluster* CR, which represents the Giant Swarm Cluster API implementation of the cluster for the AWS platform.
+
+*AWSCluster* CR object example:
+
+```
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+So, when the release 10.x.x+ in AWS installation is used, *Giant Swarm API* service creates *Cluster* and *AWSCluster* CRs in Control-Plane Kubernetes.
```

Comment:
Yeah it's probably easier to specify that this is for all Cluster API versions earlier, and state those versions. then, you won't need to qualify this so much later